Apps/FlightSoftware/Watchdog/src/i2c.c

```diff
@@ -1,295 +1,318 @@
+/*
+ * i2c.c
+ *
+ *  Created on: May 13, 2021
+ *      Author: xprize
```

Comment:
jschwartz/mschnur?

Reply:
Updated to mschnur (he wrote it all, I just helped integrate and test)